fy22_cecil_b_moore/main.py

```diff
@@ -0,0 +1,61 @@
+##Run all sql scripts in order to produce necessary tables 
+
+import os
+import psycopg2
+from config import config
+from config import config2
+#create database (i did cbm_v3 for this run)
+#create output schema
+
+#load up zip code shapefile
+os.system("shp2pgsql -s 4326 -I /Users/markmorley/Downloads/tl_Pennsylvania5Digit2009/tl_Pennsylvania5Digit2009.shp public.zips | psql -U postgres -h localhost cbm_final")
+
+
+# psql
```

Comment:
clean out old commented-out code when you're done working. Comments are OK but should be notes, not old code.